src/main/java/com/synopsys/integration/detect/tool/detector/DetectorRuleFactory.java

```diff
@@ -122,6 +122,7 @@ private DetectorRuleSet createRules(DetectDetectableFactory detectableFactory) {
         ruleSet.yield(npmShrinkwrap).to(yarnLock);
 
         ruleSet.yield(npmCli).to(pnpmLock);
+        //TODO- in at least one sample project, both lerna and pnpm were applicable.  In that case, Lerna wasn't extractable and thus Overall Status: FAILURE_DETECTOR.  Should one yield to the other?
```

Comment:
I think avoiding the conflict is better than a failure. One should yield to the other, and that should be overridable by excluding the Lerna or Pnpm `DetectorType`.